### PR TITLE
[FIX] Merge conflict resolution + getAddressFromPrivateKey for Cardano

### PR DESCRIPTION
**Pull request by tengu-br**

### CODE REVIEW

File: docs/transfers.md [hunk 1]
```diff
@@ -281,9 +281,65 @@ const transaction = await txController.createHathorTransferTransactionFromUTXO({
...
+    ]
+  })
+```
+#### `txController.createCardanoTransferTransactionFromUTXO(opts)`
+
+* `opts.inputs` (array of Input) - array of inputs to be used in the transaction.
+  * `opts.inputs[].txHash` (string) - transaction hash of the UTXO.
+  * `opts.inputs[].index` (number) - index of the UTXO output.
+  * `opts.inputs[].privateKey` (string) -  spending private key to sign the transaction with.
+* `opts.outputs[].token` (object) - optional token information (for token transactions only).
```
Comment (vmdario) [inline docs/transfers.md:316]: outputs aqui é igual o de cima né?

Comment (tengu-br) [reply docs/transfers.md]: Correto!
Atualizei para ficar de acordo no commit 01de7ad2a4171ca375e1e8f00b94e980ebca6f01

File: docs/wallets.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ console.log(mnemonic)
 #### `walletController.generateWallet(opts)`
 
 Generate a wallet for a blockchain protocol:
-* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `STELLAR`, `RIPPLE`, `HATHOR`.
+* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `STELLAR`, `RIPPLE`, `HATHOR`, `CARDANO`, `AVALANCHE`.
```
[the remark this answers is not in the document]
Comment (tengu-br) [reply docs/wallets.md]: Resolvido no commit bdb346db45d81b80fafc996b32e034c6a54dc623

File: docs/wallets.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ console.log(wallet)
 #### `walletController.generateWalletFromPrivateKey(opts)`
 
 Generate a wallet for a blockchain protocol:
-* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `STELLAR`, `RIPPLE`, `HATHOR`.
+* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `STELLAR`, `RIPPLE`, `HATHOR`, `CARDANO`, `AVALANCHE`.
```
[the remark this answers is not in the document]
Comment (tengu-br) [reply docs/wallets.md]: Resolvido no commit bdb346db45d81b80fafc996b32e034c6a54dc623

File: docs/wallets.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ const wallet = await walletController.generateWalletFromPrivateKey({
 #### `walletController.generateWalletAddressFromXpub(opts)`
 
 Generate a wallet address from xpub for a blockchain protocol:
-* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `HATHOR`.
+* `opts.protocol` (string) (__required__) - blockchain protocol supported: `BITCOIN`, `ETHEREUM`, `BSC`, `CELO`, `HATHOR`, `CARDANO`,`AVALANCHE`.
```
[the remark this answers is not in the document]
Comment (tengu-br) [reply docs/wallets.md]: Resolvido no commit bdb346db45d81b80fafc996b32e034c6a54dc623

**Pull request merged**